packages/ai-mcp/src/browser/mcp-command-contribution.ts

```diff
@@ -0,0 +1,121 @@
+
+export const StartMCPServer = {
+    id: 'mcp.startserver',
+    label: 'MCP: Start MCP Server',
+};
+export const StopMCPServer = {
+    id: 'mcp.stopserver',
+    label: 'MCP: Stop MCP Server',
+};
+
+@injectable()
+export class MCPCommandContribution implements CommandContribution {
+    @inject(AICommandHandlerFactory)
+    protected readonly commandHandlerFactory: AICommandHandlerFactory;
+
+    @inject(QuickInputService)
+    protected readonly quickInputService: QuickInputService;
+
+    @inject(MessageService)
+    protected messageService: MessageService;
+
+    @inject(MCPServerManager)
+    protected readonly mcpServerManager: MCPServerManager;
+
+    @inject(ToolInvocationRegistry)
+    protected readonly toolInvocationRegistry: ToolInvocationRegistry;
+
+    private async getMCPServerSelection(serverNames: String[]): Promise<string | undefined> {
+        if (!serverNames || serverNames.length === 0) {
+            this.messageService.error('No MCP Servers configured.');
+            return undefined;
+        }
+        const options = serverNames.map(mcpServerName => ({ label: mcpServerName as string }));
+        const result = await this.quickInputService.showQuickPick(options);
+        if (!result) {
+            return undefined;
+        }
+        return result.label;
+    }
+
+    registerCommands(commandRegistry: CommandRegistry): void {
+        commandRegistry.registerCommand(StopMCPServer, this.commandHandlerFactory({
+            execute: async () => {
+                try {
+                    const selection = await this.getMCPServerSelection(await this.mcpServerManager.getStartedServers());
+                    if (!selection) {
+                        return;
+                    }
+                    this.toolInvocationRegistry.unregisterAllTools(`mcp_${selection}`);
+                    this.mcpServerManager.stopServer(selection);
+                } catch (error) {
+                    console.error('Error while stopping MCP server:', error);
+                }
+            }
+        }));
+
+        commandRegistry.registerCommand(StartMCPServer, this.commandHandlerFactory({
+            execute: async () => {
+                try {
+                    const selection = await this.getMCPServerSelection(await this.mcpServerManager.getServerNames());
+                    if (!selection) {
+                        return;
+                    }
+                    this.mcpServerManager.startServer(selection);
+                    const { tools } = await this.mcpServerManager.getTools(selection);
+                    const toolRequests: ToolRequest[] = tools.map((tool: any) => this.convertToToolRequest(tool, selection));
+
+                    for (const toolRequest of toolRequests) {
+                        this.toolInvocationRegistry.registerTool(toolRequest);
+                    }
+                } catch (error) {
+                    console.error('Error while starting MCP server:', error);
```

Comment:
We should log a generic message like "MCP server <name> could not be started" or "An error occured when starting MCP server <name>" to the MessageService so that the user gets some feedback